spec/reporter/alb_errors_report_builder_spec.rb

```diff
@@ -116,6 +116,6 @@ def mock_get_metric_statistic(metric_name:, response:)
   end
 
   def get_report_datapoint_for_timestamp(timestamp:, data:)
-    data.find { |d| d[:label] == timestamp.getlocal.strftime('%H:%M') }
+    data.find { |d| d[:label] == timestamp.in_time_zone(TIME_ZONE).strftime('%H:%M') }
```

Comment:
@jmhooper `.getlocal` steps "down" from an `ActiveSupport::TimeWithZone` (rich timezone support) to the stdlib `Time` (which has only UTC or local time) which I think is an anti-pattern.

I almost changed all the rests to be `TIME_ZONE.parse` instead of `TIme.parse` to make sure we only keep to the "rich" timezones....but this is the minimum these specs needed to pass on my machine, so 🤷‍♂